src/panels/DevPanel/DevPanel.tsx

```diff
@@ -0,0 +1,16 @@
+import { Tabs } from '@mantine/core';
+
+import { PropertyTest } from './Subpanels/PropertyTest';
+
+export function DevPanel() {
+  return (
+    <Tabs allowTabDeactivation>
```

Comment:
Make the property test default? was a bit confusing that nothing was shown to begin with